ViewLayer/DisplayDashboardUI/RaceModeDashboardUI.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "I_RaceModeDashboardUI.h"
+
+namespace Ui
+{
+    class RaceModeDashboardUI;
```

Comment:
Theres' nothing we should do about it now, but for future reference acronyms in CapCase names shouldn't be all capitalized. 

`RaceModeDashboardUI` -> `RaceModeDashboardUI`
`PCBOverCANLockout` -> `PcbOverCanLockout`